CMSgov/mint-app
# [EASI-2400] APM Question changes

# EASI-2400

## Changes and Description

- Backend
  - Remove the alternativePaymentModel boolean
  - Add another enumerated value to AlternativePaymentModelType for "NOT_APM"
  - Update tests
  - Update Postman Collection
- Frontend
  - Update general characteristics page 2 to reflect what's in the Figma (single question for APM with 4 checkboxes)
  - Add frontend validation so that if you select "It is not an APM" all other checkboxes are unchecked and disabled. 
  - Updated General Characteristics Read Only page 



## How to test this change

1. Preseed or create new model plan
2. Navigate to page 2 of `General Characteristics` task list 
3. Ensure that the APM question is now one question, instead of two.
4. Clicking `It is not an APM` should unchecked and disable all others
5. Ensure data is saving correctly 
6. Navigate to the Read Only page and ensure the correct data is being showing up
   ![Screenshot 2022-11-30 at 2 20 27 PM](https://user-images.githubusercontent.com/2349702/204889117-22194e99-aaf9-4fb5-8c21-e45fb6c83cef.png)


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
